pyrcel/distributions.py

```diff
@@ -69,9 +67,13 @@ class Lognorm(BaseDistribution):
     Methods
     -------
     pdf(x)
-        Evaluate distribution at a particular value
+        Evaluate size distribution density dN/dx (not logarithmic) at a particular radius x
+    pdfloge(x)
+        Evaluate size distribution logarithmic density dN/dln(x) at a particular radius x
+    pdflog10(x)
+        Evaluate size distribution logarithmic density dN/dlog(x) at a particular radius x
     cdf(x)
-        Evaluate cumulative distribution at a particular value.
+        Evaluate cumulative concentration up to a particular radius.x
```

Comment:
Think there's a small typo in your edit ->

```suggestion
        Evaluate cumulative concentration up to a particular radius x
```